genie-agent/src/main/java/com/netflix/genie/agent/execution/statemachine/listeners/LoggingListener.java

```diff
@@ -93,7 +92,7 @@ public void eventNotAccepted(final Message<Events> event) {
      */
     @Override
     public void transition(final Transition<States, Events> transition) {
-        UserConsole.getLogger().info(
+        log.info(
```

Comment:
Is it intentional to not use UserConsole.getLogger()?

Reply:
Yes, this was printing the agent state machine transition changes in the user terminal.
That's a bit too low level, so I'm just sending it to the log file instead.